### PR TITLE
[Console] Document `SymfonyStyle` additions

### LINKED ISSUE #15969: [Console] `SymfonyStyle` enhancements
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#43595
| PR author(s) | @kbond
| Merged in    | 5.4

### PR DESCRIPTION
**Pull request by kbond**
Docs for https://github.com/symfony/symfony/pull/43595.

Fixes #15969.

### CODE REVIEW

**Comment by kbond** [comment]
~When I built the docs using the new system, the generated docs in `_build/output` were not styled and missing stuff so I wasn't able to confirm this looks correct...~

Edit, I was able to correct this by upgrading `symfony-tools/doc-builder` and `build.php`.

File: console/style.rst [hunk 1]
```diff
@@ -165,6 +165,14 @@ Content Methods
             ['foo4' => 'bar4']
         );
 
+:method:`Symfony\\Component\\Console\\Style\\SymfonyStyle::createTable`
+    Creates an instance of :class:`Symfony\\Component\\Console\\Helper\\Table`
+    styled according to the Symfony Style Guide.
```
Comment (kbond) [inline console/style.rst:170]: A feature of this method is the table uses the `ConsoleOutput` if available so you can `appendRows` - should I document this?

**Comment by javiereguiluz** [comment]
Thanks Kevin!

File: console/style.rst (hunk 1 above)
Comment (javiereguiluz) [reply console/style.rst:170]: Yes, I think it'd be a good idea. I added a very brief mention to "appending rows" while merging. Thanks!

**Pull request merged**